imu_sensor_broadcaster/doc/userdoc.rst

```diff
@@ -0,0 +1,18 @@
+.. _imu_sensor_broadcaster_userdoc:
```

Comment:
Can you link this file to the main documentation document (doc/index.rst)?

Reply:
I linked it